scripts/scratch_query/upload_to_bq.py

```diff
@@ -0,0 +1,57 @@
+"""Upload NDJSON file to BigQuery
```

Comment:
Today I learned that [newline-delimited JSON](http://ndjson.org/) (NDJSON) is a thing!